backend/src/main/java/com/yohoard/util/DateTimeUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.yohoard.util;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+public class DateTimeUtils {
```

Comment:
Na pewno taka klasa utilsowa jest potrzebna? W Javie 17 nie ma jakiegoś rozwiązania w kodzie out of the box?

Reply:
Nie znalazłem gotowej metody której możnaby wykorzystać przy mikrosekundach